src/test/ui/traits/alias/self-in-generics.rs

```diff
@@ -1,9 +1,9 @@
 // astconv uses `FreshTy(0)` as a dummy `Self` type when instanciating trait objects.
 // This `FreshTy(0)` can leak into substs, causing ICEs in several places.
-// Using `save-analysis` triggers type-checking `f` that would be normally skipped
-// as `type_of` emitted an error.
-//
-// compile-flags: -Zsave-analysis
+
+
+
+
```

Comment:
Please remove extra whitespace.

Reply:
I did this in order to avoid changing line numbers in the `.stderr` files. But I can change them if people feel strongly about it.

Reply:
I'd prefer to bless stderr files as it's quite unclear for future contributors why these tests have extra lines.